### DISCUSSION TITLE
Prism 8.1 Xamarin.Forms

### DISCUSSION THREAD

**Discussion by albilaga** [PrismLibrary/Prism#2898] [Xamarin.Forms]
We are upgrading from prism 7.2 to 8.1 but we are having difficulties on how to implement handling resolution errors in [here](https://prismlibrary.com/docs/dependency-injection/resolution-errors.html) ? Is there any samples on how xamarin.forms do it?

**Comment by dansiegel** [comment]
A ContainerResolutionException isn't something we really should be giving you a ton of guidance on. It is the result of your code not registering something correctly or sometimes of your code throwing initialization errors. This could be for any number of reasons. The ContainerResolutionException was introduced to help you diagnose what in your code is going wrong quicker. Many times the error message itself should be your first clue as to what is going wrong. Other times you may instead want to look at calling the GetErrors method on the ContainerResolutionException. This will help you get a more drilled down look at what dependencies may not be getting resolved.
